src/rpc/misc.cpp

```diff
@@ -345,7 +345,7 @@ static UniValue setmocktime(const JSONRPCRequest& request)
                 RPCExamples{""},
             }.Check(request);
 
-    if (!Params().MineBlocksOnDemand())
+    if (!Params().IsMockableChain())
```

Comment:
```suggestion
    if (!Params().IsMockableChain()) {
```

Might as well fix up with the new style (opening and closing {}). Not a blocking nit.

Reply:
done [e8600d9](https://github.com/bitcoin/bitcoin/pull/18263/commits/e8600d9f2d2c1b38257d91a3cf7eba670b139465)